tests/ContentRepository90/Rules/FusionNodeHiddenInIndexRector/Fixture/some_file.fusion.inc

```diff
@@ -38,7 +38,7 @@ prototype(Neos.Fusion.Form:Checkbox)  < prototype(Neos.Fusion.Form:Component.Fie
     #
     # pass down props
     #
-    attributes = ${node.property('hiddenInIndex') || documentNode.property('hiddenInIndex') || site.property('hiddenInIndex')}
+    attributes = ${node.property('hiddenInIndex') || documentNode.property('hiddenInIndex') || site.property('hiddenInIndex') || q(node).property('hiddenInIndex') || q(documentNode).property('hiddenInIndex')}
```

Comment:
but wait lol its now `hiddenInMenu` https://discuss.neos.io/t/neos-9-beta-10-release/6578

Reply:
Yes, right. But that's another issues right? I'll created a new issue:
https://github.com/neos/rector/issues/65